src/bitcoin/mod.rs

```diff
@@ -58,11 +58,12 @@ pub trait BitcoinInterface: Send {
         descs: &[descriptors::InheritanceDescriptor],
     ) -> Vec<UTxO>;
 
-    /// Get all coins that were confirmed, and at what height and time.
+    /// Get all coins that were confirmed, and at what height and time. Along with "expired"
+    /// unconfirmed coins (for instance whose creating transaction may have been replaced).
     fn confirmed_coins(
```

Comment:
Maybe rename the fn now for confirmed_and_expired_coins
or maybe we can find a way to have `confirmed_coins` and `expired_coins` sharing the same cache

Reply:
Yeah the name could be updated. I'm going to keep it as is for now since it's documented at least in the comment.

For the cache it could be nice to have but then you get into managing its size, evictions, etc.. Until it becomes a bottleneck i don't think it's worth the effort / complexity.